guides/source/getting_started.md

```diff
@@ -388,9 +388,9 @@ create and read. The form for doing this will look like this:
 ![The new article form](images/getting_started/new_article.png)
 
 It will look a little basic for now, but that's ok. We'll look at improving the
-styling for it afterwards.
+styling for it afterward.
```

Comment:
Its not american English, just a favoured way to use it. Since we use `afterwards` everywhere else, lets continue its usage for now.